_spinnaker/install.md

```diff
@@ -6,7 +6,19 @@ redirect_from:
   - /install_guide/install/
   - /install-guide/gettings_started/
 ---
-# Installing Armory Spinnaker
+# Installing Armory Spinnaker the Easy Way
```

Comment:
I don't know if I agree with this "Easy"/"Hard" way differentiation between the titles. It's catchy and Minnaker is indeed easier to install, but Minnaker's scope is meant for things like POCs, not production. My first impression as an end user is then "why does a hard way even exist if there's an easy way?" I'm also liable to never actually look at the hard way and only use the easy way ever.

I'd title the sections "Installing a Proof of Concept Deployment of Armory Spinnaker"/"Installing a Production Deployment of Armory Spinnaker" or something similar to highlight that difference. 

With that said, I think "Installing Armory Spinnaker the Easy Way" would be great for a blog post title to grab people, similar to what k8s did: https://kubernetes.io/blog/2017/11/kubernetes-easy-way/

Reply:
So we are looking at ways to make Minnaker a production ready install.  But I will change for now to get this out.